crypto/serializer/deserializer_lib.c

```diff
@@ -181,7 +181,7 @@ int OSSL_DESERIALIZER_CTX_add_extra(OSSL_DESERIALIZER_CTX *ctx,
     size_t w_prev_start, w_prev_end; /* "previous" deserializers */
     size_t w_new_start, w_new_end;   /* "new" deserializers */
     size_t count; /* Used to calculate how many were added in each iteration */
```

Comment:
I will make the same comment here as the other PR: count = 0;

Reply:
I have nothing against initializing it...  but the reasoning you present in https://github.com/openssl/openssl/pull/12492#discussion_r457778670 is flawed.

Reply:
Done